<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvanceBooking</class>
 <widget class="QDialog" name="AdvanceBooking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance Booking</string>
  </property>
  <widget class="QFrame" name="frameTravelDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="0">
      <widget class="QLabel" name="labelVehicle">
       <property name="text">
        <string>Vehicle:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDestinations">
       <property name="text">
        <string>Destinations:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelNoPassengers">
       <property name="text">
        <string>No of Passengers:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBoxNoPassengers">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditVehicle">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTravelDetails">
       <property name="text">
        <string>Travel Details:-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDepartureDateTime">
       <property name="text">
        <string>Departure Date and Time:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEditDeparture">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelArrivalDateTime">
       <property name="text">
        <string>Arrival Date and Time:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEditArrival">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditDestinations">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frameSubmission">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelAdminName">
       <property name="text">
        <string>Admin Name:       </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditAdminName">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frameCustomerDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>491</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelCustomerDetails">
       <property name="text">
        <string>Customer Details:-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFirstName">
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditFirstName">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEditLastName">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAddressLine1">
       <property name="text">
        <string>Address Line 1:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditAddressLine1">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelAddressLine2">
       <property name="text">
        <string>Address Line 2:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditAddressLine2">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelCity">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditCity">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="labelDistrict">
       <property name="text">
        <string>District:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEditDistrict">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelState">
       <property name="text">
        <string>State:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditState">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="labelPinCode">
       <property name="text">
        <string>Pin Code:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEditPinCode">
       <property name="inputMask">
        <string>000000</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelMobileNo">
       <property name="text">
        <string>Mobile No:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditMobileNo">
       <property name="inputMask">
        <string>0000000000</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelLastName">
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frameAccountDetails">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>511</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelAccountDetails">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>116</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Account Details:-</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>34</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEditPerHeadAmount">
       <property name="inputMask">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelAdvanceAmount">
       <property name="text">
        <string>Advance:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="lineEditTotalAmount">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLabel" name="labelTotalAmount">
       <property name="text">
        <string>TotalAmount:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditAdvanceAmount">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelPerHeadAmount">
       <property name="text">
        <string>PerHeadAmount:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditFirstName</tabstop>
  <tabstop>lineEditLastName</tabstop>
  <tabstop>lineEditAddressLine1</tabstop>
  <tabstop>lineEditAddressLine2</tabstop>
  <tabstop>lineEditCity</tabstop>
  <tabstop>lineEditState</tabstop>
  <tabstop>lineEditDistrict</tabstop>
  <tabstop>lineEditPinCode</tabstop>
  <tabstop>lineEditMobileNo</tabstop>
  <tabstop>lineEditDestinations</tabstop>
  <tabstop>dateTimeEditDeparture</tabstop>
  <tabstop>dateTimeEditArrival</tabstop>
  <tabstop>lineEditVehicle</tabstop>
  <tabstop>spinBoxNoPassengers</tabstop>
  <tabstop>lineEditPerHeadAmount</tabstop>
  <tabstop>lineEditTotalAmount</tabstop>
  <tabstop>lineEditAdvanceAmount</tabstop>
  <tabstop>lineEditAdminName</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdvanceBooking</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdvanceBooking</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
